test/e2e/dnslookup_test.go

```diff
@@ -331,11 +355,20 @@ func writeDNSHammerFile(path string, records map[string][]string) error {
 	return csvWriter.WriteAll(data)
 }
 
-func generateTXTrecords(n int) map[string][]string {
+func generateCNAMErecords(n int, domain string) map[string][]string {
+	records := make(map[string][]string)
+	for i := 1; i <= n; i++ {
+		subDomain := strings.ToLower(randomTxt(rand.Intn(10-1) + 1)) // nolint: gosec
```

Comment:
Presumably because of the rand? Would be good to actually comment on why it's disabled (as in, add a comment in the code).